content/updates/terraform-use-cases.md

```diff
@@ -0,0 +1,202 @@
+---
+visible: true
+draft: false
+title: "Three Terraform Use-cases You Need to Start Implementing"
+author: Matt Gowie
+date: 2024-05-13
+description: "Engineering orgs that use IaC tools like Terraform aren’t typically maximizing their leverage. This article highlights at least three uses of Terraform and IaC automation that don’t necessarily center around traditional application workload infrastructure."
+image: /img/updates/tf-use-cases.jpeg
+callout: <p>👋 <b>If you're ready to take your infrastructure to the next level, we're here to help. We love to work together with engineering teams to help them build well-documented, scalable, automated IaC that make their jobs easier. <a href='/contact'>Get in touch!</a>
+---
+
+One of the main themes we’ve noticed is that platform teams often struggle when embracing the "as code" philosophy. Sure, they'll use OpenTofu or Terraform (collectively referred to from here on out as “TF”) to manage compute and other cloud resources, but they tend to stop short of applying the same principles to _everything_ in their operational domain.
+
+The goal isn't just to automate a few things here and there; the real aim is to automate _everything_. Every single operations process should be represented as code.
+
+Why? Because when you do that, you unlock a new level of consistency and reliability in managing systems that have traditionally been afterthoughts for infrastructure-as-code.
+
+In this article, we're going to highlight some resources you might be surprised to learn can be managed:
+
+* users
+* Git repos
+* monitoring and alerting configuration
+
+Hopefully your takeaway is the same thing we preach to everyone we work with: if you want a solid platform, use IaC for _everything_.
+
+### Why Aren’t People Using Terraform for Everything?
+
+There are a few reasons why engineers might not be using Terraform to deploy resources like git repositories, user management, or monitoring. One common reason we often see is that they simply don't think to do it, as they may be more focused on implementing everything else from scratch. In the case of git repositories, engineers might view it as a chicken-and-egg problem, where they manually create repositories before deploying code, rather than automating the entire process with Terraform. Additionally, time constraints and pressure to deliver quickly can lead engineers to stick with their tried-and-tested workflows instead of exploring new approaches like using Terraform to handle it all.
```

Comment:
```suggestion
There are a few reasons why engineers might not be using Terraform to deploy resources like Git repositories, user management, or monitoring. One common reason we often see is that they simply don't think to do it, as they may be more focused on implementing everything else from scratch. In the case of Git repositories, engineers might view it as a chicken-and-egg problem, where they manually create repositories before deploying code, rather than automating the entire process with Terraform. Additionally, time constraints and pressure to deliver quickly can lead engineers to stick with their tried-and-tested workflows instead of exploring new approaches like using Terraform to handle it all.
```